src/sentry/dynamic_sampling/tasks.py

```diff
@@ -278,11 +278,16 @@ def adjust_sample_rates(
     and store it in DS redis cluster, then we invalidate project config
     so relay can reread it, and we'll inject it from redis cache.
     """
-    # We need the organization object for the feature flag.
-    organization = Organization.objects.get_from_cache(id=org_id)
+    try:
+        # We need the organization object for the feature flag.
+        organization = Organization.objects.get_from_cache(id=org_id)
```

Comment:
In case an org is not found, it might be that it has been deleted in the time between the query triggering this job and the actual execution of the job.